Root/OptimizationDump.cxx

```diff
@@ -377,13 +377,17 @@ EL::StatusCode OptimizationDump :: execute ()
   }
 
   if(!m_inputMET.empty() && !m_inputJets.empty()){
-    m_effectiveMass = VD::Meff_inclusive(in_met, in_jets, in_muons, in_electrons)/1000.;
+    
+    ConstDataVector<xAOD::JetContainer> presel_jets = VD::subset_using_decor(in_jets, VD::decor_vd_pass_Presel, 1);
+    //std::cout << in_jets->size() << " " << presel_jets.size() << std::endl;
+    m_effectiveMass = VD::Meff_inclusive(in_met, presel_jets.asDataVector(), in_muons, in_electrons)/1000.;
```

Comment:
Also, these are not sorted -- and they should be (if we use an exclusive definition!)